GETTING_STARTED.md

```diff
@@ -43,6 +24,25 @@ Fetch the latest version of `master` from `upstream` (ie. the main repo):
 
     git fetch upstream master
 
+### Instalation
+
+This project needs specific ruby/bundler versions as well as node/yarn specific versions. For a local setup you will need:
+
+* Install or change your Ruby version according to the one specified at [.ruby-version](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.ruby-version) file)
+    - To manage versions, it's recommended to use [rbenv](https://github.com/rbenv/rbenv) or [RVM](https://rvm.io/)
+* Install or change Node version according to the one specified at [.node-version](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.node-version)
+    - [nodevn](https://github.com/nodenv/nodenv) is recommended as a node version manager.
```

Comment:
The setup script will now fail with error if `nodenv` is not installed (https://github.com/openfoodfoundation/openfoodnetwork/commit/09a24b0f1f0ae98ff58a4f6e3b10a253902d5ca4).

So I think it's safe to say it's officially required:
```suggestion
* Install [nodenv](https://github.com/nodenv/nodenv) to ensure the correct [.node-version](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.node-version) is used.
```

Have you had any problems with using `nodenv`?
If `nodenv` is not a suitable solution for all environments, we could consider making the script more flexible to allow any node version manager (like we do for ruby [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/script/setup#L20-L28)).